apps/condo/domains/common/utils/tables.utils.ts

```diff
@@ -380,3 +380,22 @@ export const convertColumns = (
 export const getSorterMap: SorterMapType = (sorters) => {
     return Object.assign({}, ...sorters.map((sorter) => ({ [sorter.columnKey]: sorter.order })))
 }
+
+export const categoryToSearchQuery = (search: string, translations) => {
+    if (!search) return
+    const searchLCase = search.toLowerCase()
+    const whereIn = !translations ? [] : Object.keys(translations).filter(
```

Comment:
I think it would be better to write this without negate `translations` variable

```suggestion
    const whereIn = translations ?  Object.keys(translations)... : []
```